app/forms/DeclarationPage.scala

```diff
@@ -16,4 +16,18 @@
 
 package forms
 
-trait DeclarationPage
+import models.DeclarationType.{CLEARANCE, DeclarationType}
+import models.viewmodels.TariffContentKey
+
+trait DeclarationPage {
+  def defineTariffContentKeys(decType: DeclarationType): Seq[TariffContentKey] = Seq.empty
+
+  def defineJourneySpecificKey(decType: DeclarationType): String = "common"
+}
+
+abstract class ClearanceModifierPage extends DeclarationPage {
+  override def defineJourneySpecificKey(decType: DeclarationType): String = decType match {
+    case CLEARANCE => "clearance"
+    case _         => "common"
+  }
+}
```

Comment:
I would have liked to split the page specific logic out from the forms objects as (Maciej originally mentioned) there is not an actual one-to-one relationship between pages and these forms objects (e.g. YesNoAnswers). Due to time constraints I've had to settle for defining new objects where necessary. I'm hoping in the next tariff links ticket we can resolve this.